common/src/main/java/com/wynntils/services/statistics/type/StatisticEntry.java

```diff
@@ -4,11 +4,17 @@
  */
 package com.wynntils.services.statistics.type;
 
-public record StatisticEntry(int total, int count, int min, int max) {
-    public static final StatisticEntry EMPTY = new StatisticEntry(0, 0, 0, 0);
+public record StatisticEntry(int total, int count, int min, int max, long lastModified, long lastReset) {
```

Comment:
```suggestion
public record StatisticEntry(int total, int count, int min, int max, long lastModified, long firstModified) {
```

That the first modified field is reset when we reset statistics is kind of impied...

Reply:
And possibly it might make more sense to switch the order around for those two